### ISSUE TITLE
Bug: fields are missing unique:true constraint in data-package.json

### ISSUE THREAD

**Issue by timgdavies** [openreferral/specification#199]
Most `id` fields in [datapackage.json](https://github.com/openreferral/specification/blob/master/datapackage.json) are described with the text "Each X must have a unique identifier".

However, in most cases, this is not properly declared in datapackage.json which contains

```json
"constraints": {
   "required":true,
}
```

instead of

```json
"constraints": {
     "required":true,
     "unique":true
}
```

There is a question in this case of whether the schema or the description text should be treated as authoritative, but I would consider the intent of the field to be clear (ids MUST be unique) but that the constraint has not been properly enforced by the schema.

This feeds through into the documentation which is generated from the schema and presents:

![image](https://user-images.githubusercontent.com/342624/41784811-ae935066-7638-11e8-9176-607866393dae.png)

**Comment by timgdavies** [comment]
I've committed a fix to branch 199-unique-fields (see [diff against master here](https://github.com/openreferral/specification/compare/199-unique-fields?expand=1)).

You can also preview the docs as they appear for this branch at http://docs.openreferral.org/en/199-unique-fields/hsds/reference/#service_area

As this is a bug-fix that could affect users who are using datapackage.json to validate their data, I would suggest it should form a 1.1.1 bug fix release, and should have at least a two week comment period. 

There may also be knock on effects to the API specification schema to be considered if @kinlane's work carried this bug forward into the API Spec. I've not checked on this yet. 

----

The *alternative* work-around for this issue, if the schema 'bug' (i.e. weaker validation than the normative descriptions state) is left in until a next version because it would affect current adopters, would be that we drop the 'Unique?' column from the documentation display, and rely on the prose to cover this.

**Comment by japrogramer** [comment]
[In my previous comment](https://github.com/openreferral/specification/issues/217#issuecomment-631649412) I asked if all ids are UUIDs as suggested by
![image](https://user-images.githubusercontent.com/1387859/84338931-f55ed900-ab62-11ea-9a51-6135b9d28090.png)
And
![image](https://user-images.githubusercontent.com/1387859/84338990-18898880-ab63-11ea-96c6-8a725641acbf.png)


However this is contradicted here,
![image](https://user-images.githubusercontent.com/1387859/84339056-3eaf2880-ab63-11ea-8b09-7b9f95f572da.png)

what are overlapping identifiers? adding a prefix to a uuid defeats the purpose of a uuid in my opinion. perhaps that prefix needs to be stored in a separate field.

**Comment by mrshll1001** [comment]
Closing as this has been fixed in 3.0 (at the latest!)
